File: objectivity/player_report.m
#import "player_report.h"

@implementation PlayerReport

+ (void) about: (GameState *)level
{
  NSLog(@"%u Players", (unsigned int)[[level players] count]);
  NSLog (@"----------");

  for (Player *player in [level players])
  {
    NSLog (@"%@", [player name]);
  }
}
@end

File: cee/player_report.c
#include "player_report.h"


/* Player Report */

void player_report(GameState *game_state)
{
	int i;
	for(i = 0; i < game_state->player_position; i++)
	{
		printf("Player: %s\n", game_state->players[i]->name);
	}
}
